README.md

```diff
@@ -6,22 +6,19 @@ Rust userland library for Tock (WIP)
 
 This project is nascent and still under heavy development, but first steps:
 
-1. Ensure you have a working toolchain available:
+```bash
+./build_examples.sh
+```
 
-    `rustup install nightly-2017-12-16`
+Then load `blink` on a board:
 
-2. Get a copy of this toolchain, in this repo's root:
+```bash
+./run_example.sh blink
+```
```

Comment:
One could notice here that due to behavior of the tockloader the parameters passed to tockloader in the run_example script only work for the nrf52dk, currently. At least as far as I can judge.